### PR TITLE
amm: Small cleanups

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: contracts/amm/TempusAMM.sol [hunk 1]
```diff
@@ -307,7 +296,19 @@ contract TempusAMM is ITempusAMM, ERC20, Pausable, Ownable {
         return invariant.divDown(totalSupply());
     }
 
-    /// Query functions
```
Comment (axic) [inline contracts/amm/TempusAMM.sol:310]: These replace the natspec from the interface.

File: contracts/amm/TempusAMM.sol [hunk 2]
```diff
@@ -158,19 +158,8 @@ contract TempusAMM is ITempusAMM, ERC20, Pausable, Ownable {
...
-        returns (uint256 token0Balance, uint256 token1Balance)
-    {
-        uint256 supply = totalSupply();
-        uint256 accountBalance = balanceOf(account);
-        token0Balance = (accountBalance * token0.balanceOf(address(this))) / supply;
-        token1Balance = (accountBalance * token1.balanceOf(address(this))) / supply;
-    }
+    // Removing liquidity
 
-    /// Removing liquidity
```
Comment (axic) [inline contracts/amm/TempusAMM.sol:173]: This replaces the natspec from the interface. The other cases may not as there is an empty line in between.

File: contracts/amm/TempusAMM.sol [hunk 3]
```diff
@@ -328,12 +329,11 @@ contract TempusAMM is ITempusAMM, ERC20, Pausable, Ownable {
                 .divDown(scalingFactor);
     }
 
-    function getExpectedInGivenOut(uint256 amountOut, address tokenIn) external view override returns (uint256) {
```
Comment (axic) [inline contracts/amm/TempusAMM.sol:331]: Every other function uses `IPoolShare` and this is the only exception.

**Review by mijovic — APPROVED** [review]

**Comment by mijovic** [comment]
@axic can you rebase to solve conflicts?

**Comment by axic** [comment]
@mijovic rebased

**Pull request merged**